### PR TITLE
Updated APIs and renamed agent to chaos delagate in logs

### PR DESCRIPTION
**Pull request by SarthakJain26**
- Updated APIs and renamed agent to chaos delagate in logs
Signed-off-by: Sarthak Jain <sarthak.jain@harness.io>

### CODE REVIEW

File: Usage_interactive.md [hunk 1]
```diff
@@ -167,20 +167,20 @@ Installation Mode: cluster
...
 
-💡 Connecting agent to Chaos Center.
-🏃 Agents are running!!
+💡 Connecting chaos delegate to ChaosCenter.
+🏃 Chaos Delegates are running!!
 
-🚀 Agent Connection Successful!! 🎉
-👉 Litmus agents can be accessed here: https://cloud.chaosnative.com/agents
+🚀 Chaos Delegate Connection Successful!! 🎉
+👉 Litmus chaos delegates can be accessed here: https://cloud.chaosnative.com/agents
```
Comment (Saranya-jena) [inline Usage_interactive.md:178]: We also need to update the path from `/agents` -> `/delegates`

File: pkg/cmd/create/agent.go [hunk 2]
```diff
@@ -256,8 +256,8 @@ var agentCmd = &cobra.Command{
 		// Watch subscriber pod status
 		k8s.WatchPod(k8s.WatchPodParams{Namespace: newAgent.Namespace, Label: utils.ChaosAgentLabel}, &kubeconfig)
 
-		utils.White_B.Println("\n🚀 Agent Connection Successful!! 🎉")
-		utils.White_B.Println("👉 Agents can be accessed here: " + fmt.Sprintf("%s/%s", credentials.Endpoint, utils.ChaosAgentPath))
+		utils.White_B.Println("\n🚀 Chaos Delegate Connection Successful!! 🎉")
+		utils.White_B.Println("👉 Chaos Delegates can be accessed here: " + fmt.Sprintf("%s/%s", credentials.Endpoint, utils.ChaosAgentPath))
```
Comment (Saranya-jena) [inline pkg/cmd/create/agent.go:260]: Could you please cheack if the `utils.ChaosAgentPath` is correct?

File: pkg/cmd/get/get.go [hunk 3]
```diff
@@ -26,7 +26,7 @@ var GetCmd = &cobra.Command{
 		#get list of projects accessed by the user
 		chaosctl get projects
 
-		#get list of agents within the project
+		#get list of chaos delegates within the project
 		chaosctl get agents --project-id=""
```
Comment (Saranya-jena) [inline pkg/cmd/get/get.go:30]: Here also wee can update the command.

File: pkg/cmd/upgrade/upgrade.go [hunk 4]
```diff
@@ -23,7 +23,7 @@ import (
 var UpgradeCmd = &cobra.Command{
 	Use: "upgrade",
 	Short: `Examples:
-		#upgrade version of your agent
+		#upgrade version of your chaos delegate
 		chaosctl upgrade agent --cluster-id="4cc25543-36c8-4373-897b-2e5dbbe87bcf" --project-id="d861b650-1549-4574-b2ba-ab754058dd04" --non-interactive
```
Comment (Saranya-jena) [inline pkg/cmd/upgrade/upgrade.go:27]: Same here.

**Review by Saranya-jena — CHANGES_REQUESTED** [review]

File: pkg/cmd/create/agent.go (hunk 2 above)
Comment (SarthakJain26) [reply pkg/cmd/create/agent.go:260]: I have just updated logs, no changes in ChaosAgentPath

File: pkg/cmd/get/get.go (hunk 3 above)
Comment (SarthakJain26) [reply pkg/cmd/get/get.go:30]: Yes, this PR is to only update logs, messages and comments. I'll raise a separate PR to update the commands and its usages

**Review by Saranya-jena — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**